common/dna_data/wc_orc_dna.txt

```diff
@@ -0,0 +1,825 @@
+﻿### Orcs
+blackhand_dna = {
```

Comment:
Blackhand has too rough and fatless cheeks.
Also, his profile and chin it too forward.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/109343982-8d365b80-7887-11eb-9fb7-31962b83e106.png)

</details>

Reply:
Fixed in 5bebd28b9fcc5a0aee20ad54fce0dfcd03d2b6cb.